radicle-surf/src/diff.rs

```diff
@@ -121,6 +121,20 @@ pub enum FileDiff {
     },
 }
 
+/// Commit statistics.
```

Comment:
Also, this documentation should be changed:
```suggestion
/// Statistics describing a particular [`Diff`].
```

Reply:
Done in [3c3527c](https://github.com/radicle-dev/radicle-git/pull/11/commits/3c3527c970b08e8c10f3b5944f7c6718f846d17b)